source/_integrations/backup.markdown

```diff
@@ -60,12 +60,16 @@ This is a YAML example for an automation that initiate a backup every night
 at 3 AM:
 
 ```yaml
-automation:
-  - alias: "Backup Home Assistant every night at 3 AM"
-    trigger:
-      platform: time
-      at: "03:00:00"
-    action:
-      alias: "Create backup now"
-      service: backup.create
+- id: '1702841556346'
+  alias: Backup Home Assistant every night at 3 AM
+  description: Create Backup
```

Comment:
Description is optional and has no value for this example; adding this doesn't help.